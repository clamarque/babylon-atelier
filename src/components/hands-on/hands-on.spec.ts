import HandsOn from '@/components/hands-on/hands-on.vue';
import { shallowMount } from '@vue/test-utils';

describe('HandsOn', () => {
  it('should render the hands on component', () => {
    const wrapper = shallowMount(HandsOn);

    // Then
    expect(wrapper.html()).toMatchSnapshot();
  });
});
