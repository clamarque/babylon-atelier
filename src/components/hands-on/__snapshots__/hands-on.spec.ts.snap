// Vitest Snapshot v1

exports[`HandsOn > should render the hands on component 1`] = `
"<div class=\\"hands-on\\">
  <h1 class=\\"green\\">Welcome!</h1>
  <div class=\\"scene-display-container\\">
    <scene-display-stub></scene-display-stub>
  </div>
</div>"
`;
